src/main/java/org/gridsuite/filter/server/FilterController.java

```diff
@@ -55,16 +55,18 @@ public FilterController(FilterService service) {
             .orElse(ResponseEntity.notFound().build());
     }
 
-    @PostMapping(value = "filters/", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @PostMapping(value = "/filters/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
     @Operation(summary = "Create a filter")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The filter has been successfully created")})
-    public ResponseEntity<AbstractFilter> createFilter(@RequestBody(required = true) AbstractFilter filter) {
+    public ResponseEntity<AbstractFilter> createFilter(@PathVariable("id") UUID filterId,
+                                                       @RequestBody AbstractFilter filter) {
+        filter.setId(filterId);
         return ResponseEntity.ok()
             .contentType(MediaType.APPLICATION_JSON)
             .body(service.createFilter(filter));
     }
 
-    @PutMapping(value = "filters/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @PutMapping(value = "/filters/{id}", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Is this a fix of something ? Does it need another PR for it ?

Reply:
no need to do a PR just for that, it's a small fix and I need it anyway to make my PR work